Wiley Online Library.js

```diff
@@ -229,7 +229,14 @@ function scrapeBibTeX(doc, url, pdfUrl) {
 				'&fileFormat=REFERENCE_MANAGER' +
 				'&hasAbstract=CITATION_AND_ABSTRACT';
 	ZU.doPost(postUrl, body, function(text) {
-		//Z.debug(text)
+		//on at least one occasion, Wiley puts RIS within RIS, thus causing havoc.
```

Comment:
You mean, Wiley puts metadata of related article into the RIS of the article itself? I would add "related articles" somewhere in the comment here...